solutions/cf-702C.mdx

```diff
@@ -354,7 +354,12 @@ public class CellularNetwork {
 		int r = 0;
 		for (int i = 0; i < n; i++) {
 			int distance = Integer.MAX_VALUE;
-			//The Lower() and Higher() Methods of the TreeSet class returns a value of the Type Integer here and null if there is no such element.
+				      
+			/*
+			   The Lower() and Higher() Methods of the TreeSet class returns a value of Type
+			   Integer here and null if there is no such element.
+			*/
+				      
```

Comment:
```suggestion
			/*
			 * .lower() and .higher() methods return a value of type
			 * Integer if the value exists and null if it doesn't
			 */
```

Reply:
Forgot the stars my bad 😅. Please let me know if you want me to make any more changes